BrainCloudClient/Assets/BrainCloud/Client/BrainCloud/Internal/BrainCloudComms.cs

```diff
@@ -903,7 +903,26 @@ private void HandleResponseBundle(string jsonData)
                 }
                 return;
             }
-
+            
+            if (!IsJsonValid(jsonData))
+            {
+                _cachedReasonCode = ReasonCodes.CLIENT_NETWORK_ERROR_TIMEOUT;
```

Comment:
Does timeout make sense here? We could use an INVALID URL code if we have one. 🤔

Reply:
Or PARSING ERROR.

Reply:
I attempted to find a URL one but didnt find one. Parsing Error would be good, I found a JSON_PARSING_ERROR that I will replace with